e2e-tests/src/test/high_latency.rs

```diff
@@ -0,0 +1,86 @@
+use std::cmp::max;
+
+use aleph_client::{
+    pallets::session::SessionApi,
+    waiting::{AlephWaiting, BlockStatus},
+};
+use log::info;
+
+use crate::{config::setup_test, synthetic_network::set_out_latency};
+
+#[tokio::test]
+pub async fn high_out_latency() -> anyhow::Result<()> {
+    let config = setup_test();
+
+    let connections = config.create_signed_connections().await;
+    info!("waiting for at least session 3");
+    for connection in &connections {
+        if connection.get_session(None).await < 3 {
+            connection.wait_for_session(3, BlockStatus::Finalized).await;
+        }
+    }
+
+    info!("setting out-latency");
+    for validator in config.validator_names() {
+        info!(
+            "setting out-latency of node {} to {}",
+            validator, config.test_case_params.out_latency
+        );
+        set_out_latency(config.test_case_params.out_latency, &validator).await;
+    }
+
+    let mut max_session = 0;
+    for connection in &connections {
+        let node_session = connection.get_session(None).await;
+        max_session = max(max_session, node_session);
```

Comment:
This suggests that the current session can be different for different nodes. Why does this happen?

Reply:
Some can be on `+1` already?

Reply:
Anyway, one node can be even on arbitrary low round. Actually, that suggest that this test can be made less flaky/different - let them reach a session -> disconnect them so they stop proceeding -> set latency -> reconnect.